### ISSUE TITLE
"We're no longer updating this content regularly" banner is redundant on archived blogs

### ISSUE THREAD

**Issue by SamB** [MicrosoftDocs/feedback#3369]
**Is your feature request related to a problem? Please describe.**
Yes. Every time I open an archived blog post, there's a huge, unnecessary banner at the top. For example, 
<https://docs.microsoft.com/en-us/archive/blogs/ricom/non-properties-of-floating-point-numbers> looks like:
![Screenshot_20210107-135024_Chrome](https://user-images.githubusercontent.com/13903/103932725-467e7d80-50f0-11eb-90cf-7792386e6d34.png)

Why yes, it's an old blog post on an archived blog. I am aware that it's not going to be updated with new content.

**Describe the solution you'd like**
Remove the banner from the blog archives. It adds nothing and wastes a lot of space.

**Describe alternatives you've considered**
It *would* be useful to have a banner linking to the blog's new home, if the author kept it going after leaving Microsoft. But it would have to use a new banner, because the "preferred version" one almost always has uselessly-generic links, so I only rarely hover those even  when I'm on my PC.

**Additional context**
I'm not really sure if this should be a "feature request", but the "bug report" template wanted steps to reproduce for some reason.

**Comment by gewarren** [comment]
@kexugit Can you help route this issue?

**Comment by kexugit** [comment]
I don't believe the banner "adds nothing". It tells users about the status of the archived content. But I do understand the desire to be able to dismiss it. A new banner feature which allows dismissing the banner will be deployed to the archived content in a couple of months.
